.env

```diff
@@ -1,2 +1,3 @@
-STAGE='dev'
-STAC_API_INTEGRATION_API_ARN=''
\ No newline at end of file
+STAGE='test'
+DB_ALLOCATED_STORAGE=20
+STAC_API_INTEGRATION_API_ARN='arn:aws:execute-api:us-west-2:532321095167:hqprpfkrqg/*/$default'
```

Comment:
I think this was added after approval, but can this be moved to github env? The public can still see .env files

Reply:
@jjfrench this is not used, we're solely relying on github env/secrets https://github.com/MAAP-Project/maap-eoapi/blob/007d514cc86a5f4dca72d76e3c0fea1aac097818/.github/workflows/deploy.yml#L47. 

I am going to delete this, along with `deploy.sh`.